airbyte-integrations/connectors/source-oracle-strict-encrypt/src/test-integration/java/io/airbyte/integrations/source/oracle_strict_encrypt/OracleStrictEncryptJdbcSourceAcceptanceTest.java

```diff
@@ -21,6 +21,7 @@
 import io.airbyte.integrations.source.jdbc.AbstractJdbcSource;
 import io.airbyte.integrations.source.jdbc.test.JdbcSourceAcceptanceTest;
 import io.airbyte.integrations.source.oracle.OracleSource;
+import io.airbyte.integrations.source.relationaldb.RelationalDbQueryUtils;
```

Comment:
Similar to all other comments, this would be a nice win to have the `tableNameProcessed` not have to be on an additional line because we need to call `RelationalDbQueryUtils.enquoteIdentifier` instead of `enquoteIdentifier`